src/transform/phenopacket_builder.rs

```diff
@@ -1748,6 +1752,7 @@ mod tests {
     }
 
     #[rstest]
+    #[ignore]
```

Comment:
Can you use the `skip_in_ci` macro here? Then, they will at least occasionally run locally.